Akolosova/hw6

alisaorange opened the pull request.

alisaorange committed:
hw8

alisaorange committed:
hw6

dshemin commented on app/fpm/Dockerfile:
```diff
@@ -0,0 +1,10 @@
+FROM php:8.2-fpm
```
У Вас консольное приложение. Ему FPM не нужен.

dshemin commented on app/fpm/Dockerfile:
```diff
@@ -0,0 +1,10 @@
+
+WORKDIR /var/www/application.local
+
+CMD ["php-fpm"]
```
Вместо запуска FPM нужно сделать контейнер с сервером, который при старте сразу запустится.

dshemin commented on app/www/.gitignore:
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Если Вы в корневом .gitignore указали папку без '/' перед ней, то она будет игнорироваться во всех подпапках. Можно не повторять.

alisaorange committed:
чтобы сервер запускался вместе с контейнером пробовала добавить CMD ["php", "-S", "0.0.0.0:8000", "-t", "/var/www/application.local"] в Dockerfile, также вместо него добавляла
    command: php /var/www/application.local/app.php server в docker-compose.yaml , в обоих случаях одинаковая ошибка.
